borg/testsuite/repository.py

```diff
@@ -221,6 +221,53 @@ def segments_in_repository():
         assert segments_in_repository() == 2
 
 
+class RepositoryAuxiliaryCorruptionTestCase(RepositoryTestCaseBase):
+    def setUp(self):
+        super().setUp()
+        self.repository.put(b'00000000000000000000000000000000', b'foo')
+        self.repository.commit()
+        self.repository.close()
+
+    def do_commit(self):
+        with self.repository:
+            self.repository.put(b'00000000000000000000000000000000', b'fox')
+            self.repository.commit()
+
+    def test_corrupted_hints(self):
+        with open(os.path.join(self.repository.path, 'hints.0'), 'ab') as fp:
```

Comment:
do we use fp at other places? reminds me of file pointer, like in C, but in python, it is just a file (object), so f reads more naturally.